test/openAPI/features/support/helpers/helpers.js

```diff
@@ -45,6 +45,34 @@ module.exports = {
       },
     ],
   },
+  // applicant_tasks
+  applicantTasksEndpoint: 'tasks',
+  applicantTasksResponseSchema: {
```

Comment:
This response schema is used twice in the API spec, you can put it now to the `shares` section or remember about it while working on tests for  /tasks/{taskId} endpoint ;)